src/connection.js

```diff
@@ -77,31 +91,19 @@ class Connection extends EventEmitter {
         textsize: DEFAULT_TEXTSIZE,
         trustServerCertificate: true,
         useColumnNames: false,
-        useUTC: true
+        useUTC: true,
+        language: DEFAULT_LANGUAGE,
+        dateFormat: DEFAULT_DATEFORMAT
       }
     };
 
+    self.config = _.merge(defaults, config);
```

Comment:
This could be done with list of vailable config options and, if config option is not in the list throwing notice?